src/store/index.js

```diff
@@ -221,6 +232,7 @@ export default createStore({
           highlight: false,
           hide: false,
           className: bot.getClassname(),
+          promptId: id,
```

Comment:
```suggestion
          id,
```

Provide an uuid for all messages is good for future. Since all messages have an `id`, it is no need to specify `prompt`.

Reply:
do you mean you want to have an id for response message as well?

something like
```js
          id: uuidv4(),
          promptId: id, // assign prompt `id` to response message, to get the prompt message content for resending
```

Reply:
Oh, I made a mistake. Forget this.